src/app/app.component.html

```diff
@@ -948,6 +948,10 @@ <h4 style="text-align: center;">
         [showRefLines]="showRefLines"
         [showRefLabels]="showRefLabels"
         [referenceLines]="refLines"
+        [showMarkers]="showMarkers"
+        [showMarkerLabels]="showMarkerLabels"
+        [markers]="markers"
+        [markerColors]="markerColors"
```

Comment:
These input options should be documented for every chart using it.

Reply:
Documented usage of markers to line-chart component. Other components should be validated by ones who know them better, help needed :) ...I'm still unfamiliar of the library as whole.

Reply:
Don't worry mate, it should be done by someone who has wider knowledge and experience on the library as you said.

Good job btw.

Reply:
Thanks for help in review